go.sum

```diff
@@ -2081,25 +2074,23 @@ github.com/pmezard/go-difflib v1.0.0/go.mod h1:iKH77koFhYxTK1pcRnkKkqfTogsbg7gZN
 github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2 h1:Jamvg5psRIccs7FGNTlIRMkT8wgtp5eCXdBlqhYGL6U=
 github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2/go.mod h1:iKH77koFhYxTK1pcRnkKkqfTogsbg7gZNVY4sRDYZ/4=
 github.com/pquerna/cachecontrol v0.0.0-20171018203845-0dec1b30a021/go.mod h1:prYjPmNq4d1NPVmpShWobRqXY3q7Vp+80DqgxxUrUIA=
-github.com/prometheus/client_golang v1.17.0 h1:rl2sfwZMtSthVU752MqfjQozy7blglC+1SOtjMAMh+Q=
-github.com/prometheus/client_golang v1.17.0/go.mod h1:VeL+gMmOAxkS2IqfCq0ZmHSL+LjWfWDUmp1mBz9JgUY=
+github.com/prometheus/client_golang v0.9.1/go.mod h1:7SWBe2y4D6OKWSNQJUaRYU/AaXPKyh/dDVn+NZz0KFw=
+github.com/prometheus/client_golang v1.0.0/go.mod h1:db9x61etRT2tGnBNRi70OPL5FsnadC4Ky3P0J6CfImo=
```

Comment:
Should we be concerned that these versions, which are affected by the CVE[ fixed by the removed replace directive](https://github.com/openshift/installer/commit/598fde1dc1f94d7e0f36f446f2ef21c7bccec647), are now appearing in go.sum?

Reply:
That's a good question.
I would have assumed that since 1.19.0 is the version actually vendored, we cannot be affected.
But it wouldn't surprise me if there are vulnerability scanning tools that look at `go.sum`. @r4f4 do you know for sure?
We can add a replace directive pointing to 1.19.0 but it's only a matter of time until we're managing all dependencies manually if we can't ever clean these up.

Reply:
> but it's only a matter of time until we're managing all dependencies manually if we can't ever clean these up.

I--for one--welcome our new cve-scanning overlords.

Reply:
I agree, let's try it and see.